pkg/ccl/streamingccl/streamingest/alter_replication_job.go

```diff
@@ -25,13 +26,14 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/cockroachdb/cockroach/pkg/util/hlc"
+	"github.com/cockroachdb/cockroach/pkg/util/timeutil"
 	"github.com/cockroachdb/errors"
 )
 
 const alterReplicationJobOp = "ALTER TENANT REPLICATION"
 
-var alterReplicationJobHeader = colinfo.ResultColumns{
-	{Name: "replication_job_id", Typ: types.Int},
+var alterReplicationCutoverHeader = colinfo.ResultColumns{
+	{Name: "cutover_time", Typ: types.TimestampTZ},
```

Comment:
I think we lose precision here if we had a logical component to the cutover timestamp.

Reply:
interesting, thanks. I did not think about that.. we're also losing nanos when cutting over to a time from SHOW. I might nag you tomorrow about this if I don't have a better way.

Reply:
I guess we have at least a couple of options:
1. print a not-so-pretty hlc such as this:

```
root@127.0.0.1:26257/defaultdb> SELECT cluster_logical_timestamp();

    cluster_logical_timestamp
----------------------------------
  1671493837007535000.0000000000
(1 row)
```
2. print a less accurate timestamp (current pr) without logical and without nanos:
```
root@127.0.0.1:26257/defaultdb> alter tenant dest5 complete replication to latest;

          cutover_time
---------------------------------
  2022-12-06 00:25:02.098763+00
(1 row)
```
similar to:
```
root@127.0.0.1:26257/defaultdb> SELECT hlc_to_timestamp(cluster_logical_timestamp());

        hlc_to_timestamp
---------------------------------
  2022-12-19 23:50:55.100638+00
(1 row)
```

I was thinking that we need something that users can understand and not hlc.

in which scenarios are we going to have issues because of accuracy?

Reply:
> in which scenarios are we going to have issues because of accuracy?

I think the main use of this time in a context where the accuracy may matter is if you are using it to do comparisons using AOST queries or crdb_internal.fingeprint or similar.  In that context, the loss of accuracy is likely OK provided that the timestamp printed is inside the GC window on both sides and if we are inaccurate in the correct direction.

Reply:
updated, PTAL.
pretty time for replication and retained times, hlc for cutover.